crates/pypi_modifiers/src/pypi_tags.rs

```diff
@@ -159,10 +160,20 @@ fn get_implementation_name(python_record: &PackageRecord) -> miette::Result<&'st
     }
 }
 
+/// Return whether the specified record has gil disabled (by being a free-threaded python interpreter)
+/// by looking into the build string of the record.
+fn gil_disabled(python_record: &PackageRecord) -> miette::Result<bool> {
```

Comment:
I'm not sure we can depend on that `t` I've asked in the conda-forge zulip: https://conda-forge.zulipchat.com/#narrow/channel/457337-general/topic/Python.20freethreading.20from.20record/near/490528853

Also, I think you can skip the result

Reply:
I agree. Let's wait for the resolution from their side?

Reply:
based on discussion with @wolfv - it seems right way to do things, we just need to additional check if it's from conda-forge

Reply:
Didnt follow the discussion with wolf but isnt isurufs answer on zulip the rught approach for the tags? 

> Then `python_abi` is the correct way. You don't have to check that it ends with `t`. Just use the entirety of `cp313t`